app/views/groups/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<header>
+  <div class="title">
+    <div></div>
+    <div><h2>Categories</h2></div>
+    <div><%= link_to 'New', new_group_path %></div>
+  </div>
+
+  <%= render partial: 'shared/nav' %>
+</header>
+
+<div class="container">
```

Comment:
- [ ] You did a great job on the functionalities but you need to fix the below style issues to follow the given design.
  - Please make sure the button for adding the new category and transaction should be aligned at the bottom, Currently, It's at the top right corner. ( Consider the Scan button as add new button from the given design).
  - Please make sure the add new button follows the styles from the design example: colors, etc.
  
### Your design

![image](https://github.com/anthonymr/buddget/assets/83803414/7f8b6e9e-512d-45fd-bd2b-652b84b81547)

### Given design

![image](https://github.com/anthonymr/buddget/assets/83803414/6cd0386c-b16c-40e0-a678-e23595ab88e7)